strongbox-web-core/src/main/java/org/carlspring/strongbox/web/HeaderMappingFilter.java

```diff
@@ -57,6 +59,7 @@
     private static final String USER_AGENT_UNKNOWN = "unknown";
     private static final String USER_AGENT_NUGET = "NuGet";
     private static final String USER_AGENT_MAVEN = "Maven";
+    private static final String USER_AGENT_NPM= "npm";
```

Comment:
I think it would probably be better to have this as uppercase.

Reply:
`npm` is lower case. For example this real user agent header:
`user-agent = npm/3.5.2 node/v4.2.6 linux x64`

Reply:
Okay, in this case I guess, we should leave it as it is.